lib/livebook_web/live/session_live/popout_window_live.ex

```diff
@@ -0,0 +1,141 @@
+defmodule LivebookWeb.SessionLive.PopoutWindowLive do
+  use LivebookWeb, :live_view
+
+  alias Livebook.{Notebook, Session, Sessions}
+  alias Livebook.Notebook.Cell
+
+  @impl true
+  def mount(%{"id" => session_id}, _session, socket) do
+    {:ok, session} = Sessions.fetch_session(session_id)
+
+    data = Session.get_data(session.pid)
+
+    Session.subscribe(session_id)
+
+    {:ok,
+     socket
+     |> assign(
+       session: session,
+       # TODO
+       client_id: "",
```

Comment:
What is the best way to fetch the client_id from the main window?

Reply:
Each process needs to register as a separate client, so we need to register this LV and it will have its own client_id.